lib/code_corps/github/sync/sync.ex

```diff
@@ -125,37 +129,13 @@ defmodule CodeCorps.GitHub.Sync do
   defp sync_step({:ok, _} = result, _step), do: result
   defp sync_step({:error, _ = error}, _step), do: {:error, error}
 
-  @spec mark_repo(GithubRepo.t, String.t, Keyword.t) :: {:ok, GithubRepo.t} | {:error, Changeset.t}
-  defp mark_repo(%GithubRepo{} = repo, sync_state, opts \\ []) do
-    params = build_sync_params(sync_state, opts)
+  @spec mark_repo(GithubRepo.t, String.t, map) :: {:ok, GithubRepo.t} | {:error, Changeset.t}
+  defp mark_repo(%GithubRepo{} = repo, sync_state, params \\ %{}) do
     repo
-    |> GithubRepo.update_sync_changeset(params)
+    |> GithubRepo.update_sync_changeset(params |> Map.put(:sync_state, sync_state))
```

Comment:
Instead of accepting a map as the second argument here, then doing this, we were accepting a `Keyword` list and then passed those options into a pipe of functions to convert it to a map.

I was not seeing the benefit, other than an ever so slightly nicer looking function call.